DMCompiler/Program.cs

```diff
@@ -16,22 +18,29 @@ class Program {
         private static bool TryParseArguments(string[] args, out DMCompilerSettings settings) {
             settings = new();
             settings.Files = new List<string>();
+            
+            bool skipBad = false;
 
             foreach (string arg in args) {
                 switch (arg) {
                     case "--suppress-unimplemented": settings.SuppressUnimplementedWarnings = true; break;
                     case "--dump-preprocessor": settings.DumpPreprocessor = true; break;
                     case "--no-standard": settings.NoStandard = true; break;
                     case "--verbose": settings.Verbose = true; break;
+                    case "--skip-bad-args": skipBad = true; break;
                     default: {
                         string extension = Path.GetExtension(arg);
 
                         if (!String.IsNullOrEmpty(extension) && (extension == ".dme" || extension == ".dm")) {
                             settings.Files.Add(arg);
                             Console.WriteLine($"Compiling {Path.GetFileName(arg)}");
                         } else {
-                            Console.WriteLine($"Invalid arg '{arg}'");
-                            return false;
+                            if(skipBad || args.Contains("--skip-bad-args")) {
```

Comment:
The `skipBad` variable can be removed if you're just going to end up doing `args.Contains("--skip-bad-args")` anyways.

Reply:
That or initialize `skipBad` to be the Contains call